docs/source/API/core/view/memoryTraits.rst

```diff
@@ -0,0 +1,83 @@
+``MemoryTraits``
+================
+
+.. role:: cppkokkos(code)
+   :language: cppkokkos
+
+Header File: ``<Kokkos_MemoryTraits.hpp>``
+
+Usage
```

Comment:
Don't have a usage section, move examples to the end -- see https://kokkos.org/kokkos-core-wiki/templates/class_api.html